resources/css/filament/admin/theme.css

```
@@ -1445,6 +1468,29 @@
+ * Placed after that rule deliberately: `.fi-brandmark-custom img` and
+ * `.fi-brandmenu-item-mark img` have equal specificity (0,1,1), so this only
+ * wins on source order. Moving it earlier in the file silently re-enables the
+ * white-silhouette filter.
+ * ------------------------------------------------------------------------- */
+.fi-brandmark-custom {
+  @apply bg-transparent;
+}
+
+.fi-brandmark-custom img {
+  @apply w-full h-full;
+  object-fit: contain;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)